apps/modernization-ui/src/components/Table/Table.tsx

```diff
@@ -191,7 +191,7 @@ export const TableComponent = ({
                               <tr key={row}>
                                   {item.tableDetails.map((detail: TableDetail, column: number) =>
                                       detail.title ? (
-                                          detail.title === 'Not available yet' ? (
```

Comment:
This was displaying the `Not available yet` fields in a certain way without evaluating anything else on the detail.  If we do not have to support displaying `Not available yet` any more this check can be removed.  `No data` should be displayed by this component automatically if the title is not present.